network/network_linux.go

```diff
@@ -243,14 +243,24 @@ func isGreaterOrEqaulUbuntuVersion(versionToMatch int) bool {
 	return false
 }
 
-func readDnsInfo(ifName string) (DNSInfo, error) {
-	var dnsInfo DNSInfo
+func readDNSInfo(ifName string) (DNSInfo, error) {
+	var (
+		dnsInfo DNSInfo
+		out     string
+		err     error
+	)
```

Comment:
predeclaring variables (unnecessarily) is an antipattern
```suggestion
	var dnsInfo DNSInfo
```